data/card_templates.yaml

```diff
@@ -0,0 +1,37 @@
+---
+card_templates:
```

Comment:
Can you explain a little more what this `card_templates.yaml` is for and how it works?
I saw that you reference it in the macros in `mkdocs.yml`.

Reply:
Yeah, was my first idea. to use a yaml data file as source to create a MD file. 

But i maybe found something more interesting. if you check the `main.py` file in the root of the project. 

it is some python code that will search all the yaml in ulm_template dir. and will check if the yamls have a `doc:` part. If so it will be added to a jinja variable which i can use to loop over it. (see card_list.md)

macros module: https://mkdocs-macros-plugin.readthedocs.io/en/latest/macros/

mkdocs macros allows you to do jinja2 magic inside a md file!